che-incubator/chectl
# feat: introduce --delete-all flag to delete both Eclipse Che and Dev …

…Workspace related resources

Signed-off-by: Anatolii Bazko <abazko@redhat.com>



### What does this PR do?
Introduce `--delete-all` flag to delete both Eclipse Che and Dev Workspace related resources

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

https://issues.redhat.com/browse/CRW-3203

### How to test this PR?

```
./run server:deploy --platform openshift
./run server:delete --delete-all
```
Ensure there no anymore Dev Workspace resources on the cluster

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
